package.json

```diff
@@ -147,79 +148,66 @@
         "configurationAttributes": {
           "launch": {
             "required": [
-              "cwd",
-              "startupClass"
+              "mainClass"
             ],
             "properties": {
-              "cwd": {
+              "projectName": {
                 "type": "string",
-                "description": "Current working directory (Defaults to workspace root directory).",
-                "default": "${workspaceRoot}"
+                "description": "Name of the project that contains the main class (Optional, only required if the main class is contained in multiple projects)",
+                "default": ""
               },
-              "startupClass": {
+              "mainClass": {
                 "type": "string",
-                "description": "Startup class (Should be full qualified name e.g. org.eclipse.xxx.MainClass)",
+                "description": "The fully qualified name of the debuggee application's main class (e.g. org.eclipse.xxx.MainClass).",
                 "default": ""
               },
-              "classpath": {
+              "programArguments": {

```

Comment:
Can you show me an example of how to use this one?

Reply:
See the main method of java program `public static void main(String[] args)`.   `programArgument` represents the args array for java program. 
 {
            "type": "java",
            "name": "Java Debug",
            "request": "launch",
            "mainClass": "com.microsoft.debug.HelloWorld",
            "programArguments": "10 11",
            "sourcePath": [
                "${workspaceRoot}"
            ],
            "projectName": "HelloWorld"
        }